src/status_im/contexts/wallet/send/events.cljs

```diff
@@ -491,26 +489,48 @@
                                             {:event :wallet/stop-get-suggested-routes
                                              :error error}))}]}))
 
-(rf/reg-event-fx :wallet/handle-suggested-routes
+(defn- bridge-amount-greater-than-bonder-fees?
+  [{{token-decimals :decimals} :from-token
+    bonder-fees                :tx-bonder-fees
+    amount-in                  :amount-in}]
+  (let [bonder-fees      (utils.money/token->unit bonder-fees token-decimals)
+        amount-to-bridge (utils.money/token->unit amount-in token-decimals)]
+    (> amount-to-bridge bonder-fees)))
```

Comment:
I'm checking on the  `amount-in` value (because it sounds like the "amount in" the bridge). Is it right? should I use `amount-out` instead?

CC: @briansztamfater

Reply:
`amount-in` is correct 👍